src/gui/Clipboard.cpp

```diff
@@ -99,7 +99,8 @@ void Clipboard::clearCopiedText()
 
     if (m_lastCopied == clipboard->text(QClipboard::Clipboard)
         || m_lastCopied == clipboard->text(QClipboard::Selection)) {
-        setText("", false);
```

Comment:
May want to leave this as a fallback on systems where `clear()` doesn't work. https://github.com/keepassxreboot/keepassxc/issues/6101#issuecomment-1376948784

Reply:
I'm afraid that if I do that we will still have the problem with the exit hanging. There is no supported system that doesn't support clear() at this point

Reply:
I tested this and the delay on quit was still present if I included the `setText("", false);` line before the clear calls. So I will not be keeping this line in as it would defeat the whole purpose of this PR>